Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
I think this is part of the problem for and related to #798 (how to drive F# adoption). F# is a really powerful scripting language, but even the people building the compiler can't make it work for them to use it for automation. Why is that? I have no idea. IMHO it is heavily underused for scripting.

Wouldn't this new CoreCLR world be a nice chance to get a good scripting story for F#. We should really start here at the compiler, because here we can move the big blocking stones (And show the world how useful it really is).

I'm not saying that we need to use FAKE, but FAKE really shows how hard it is to do F# scripting today (always assuming no F# is installed on the machine, which is the 'interesting' case anyway -> what needs to be done to start from scratch -> how can we bundle everything or do it the CoreCLR way)